posthog.go

```diff
@@ -189,6 +189,11 @@ func (c *client) Enqueue(msg Message) (err error) {
 			if err != nil {
 				c.Errorf("unable to get feature variants - %s", err)
 			}
+
+			if m.Properties == nil {
```

Comment:
great catch, thanks @zaynetro , could you please also add a testcase for sendFeatureFlags in capture?